### ISSUE TITLE
Content/Design changes for Iowa Now - Part 2

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#6092]
- [x] Update featured resources on the Iowa Now homepage to promote the ones that are most highly trafficked on uiowa.edu.
    - [x] Self Service
    - [x] Campus Directory
    - [x] Jobs.uiowa.edu
    - [x] Not sure on fourth - @mkyoder 

- [x] Create pages and add footer links based on the footer links on the current Iowa Now site
    - [x] About
    - [x] Contact
    - [x] Submissions
    - [x] More Sources

- [x] Main Menu
    - [x] Add Faculty/Staff Experts page
    - [x] Possibly staff directory ( not sure if they want this, this is just something I am throwing out here since their existing site has a page at /staff-list
    - [x] Possibly feature topics (tags) similar to current site or like https://news.wisc.edu/science-technology/

- [x] Make sure more links for sections on homepage are in place and setup to link to the appropriate pages
  - [x] Featured Headlines - View more - Is this the page we want to link to?
  - [x] UI Spotlights - View more - link to where?
  - [x] Achievements - View more - link to where?
  - [x] In the news - View more - link to where?
# Additional Info

- ~Tricia is going to have some conversations internally about the events section.~ This is done and site updated accordingly.
- They still need to decide what do to with the "Around campus" section, pull in feeds or create custom articles?

**Comment by mkyoder** [comment]
- [x] Changed featured resources to promote ones most highlighted on uiowa.edu. Found fourth was Office365 and email and added this.
- [x] Created pages and added footer links based on current Iowa Now site. 

Would love your insight and thoughts on these @bspeare I took the content from each page and tried to format it differently so that it's not basic text on a page like how the current now.uiowa.edu site is.

**Comment by mkyoder** [comment]
**Question on the Faculty/Staff Experts page before I create it.** 

This will be a basic page with no filtering capabilities like the people directory here for example (https://tippie.uiowa.edu/people/)? The current page (https://now.uiowa.edu/experts-list) has filtering options, so I just wanted to make sure.

**Question on the staff directory page before I create it.** 

Since this page has a staff list (https://now.uiowa.edu/staff-list) almost identical (with the exception of some staff) to this page (https://osc.uiowa.edu/contact-us), do I need to reach out to the Iowa Now team to see if that page is even needed? I think the thought would be that this staff directory page on Iowa Now would link to the OSC site and avoid duplicating person profiles, but I could be wrong here.

**Comment by bspeare** [comment]
@mkyoder We did build an experts page but it looks currently broken: https://now.prod.drupal.uiowa.edu/experts-list.  I'm going to pull it down and see what the issue is.

**Comment by mkyoder** [comment]
@bspeare Good to know! Glad I asked. :)

**Comment by bspeare** [comment]
@mkyoder the new pages that you created look great!

**Comment by briand44** [comment]
@mkyoder it probably would be good to reach out about the staff page to see where we landed on that. Early on they wanted it but I vaguely remember hearing about some conversations related to linking to OSC page. I don't think I was part of those conversations though so I was assuming it was still needed since I wasn't told otherwise.

**Comment by mkyoder** [comment]
@briand44 Sounds good! I will reach out and ask and get clarification here.

**Comment by mkyoder** [comment]
Reached out to Tricia and Jack about the staff directory page. Waiting on response.

**Comment by mkyoder** [comment]
Removed lorem ipsum from the following pages:

https://now.prod.drupal.uiowa.edu/about
https://now.prod.drupal.uiowa.edu/contact
https://now.prod.drupal.uiowa.edu/submissions
https://now.prod.drupal.uiowa.edu/more-sources
https://now.prod.drupal.uiowa.edu/resources

Copy on these pages are directly from the current Iowa Now site. With the exception of this page https://now.prod.drupal.uiowa.edu/contact. The only thing different here is that I just pulled in the 'Share Your News' webform to fill this page out more visually and seems like a logical place to pull in the webform.

**Comment by mkyoder** [comment]
For these remaining items:

 Featured Headlines - View more - Is this the page we want to link to?
 UI Spotlights - View more - link to where?
 Achievements - View more - link to where?
 In the news - View more - link to where?

My question is: Is this something where I need to reach out to the Iowa Now team and ask where they want these links going to? Just wanted to make sure I wasn't missing anything else here on my end.

**Comment by bspeare** [comment]
I think we want those to link to designed overview pages like https://news.wisc.edu/science-technology/.   Do you want to try and put together a page like that and then we can just clone it and adjust the tags for each one?

**Comment by briand44** [comment]
Maybe we should chat about ^^ quick?

**Comment by bspeare** [comment]
Chat at 10:30 in Slack?

**Comment by briand44** [comment]
Sure, If I have problems with Slack we can move to a quick zoom

**Comment by mkyoder** [comment]
What do we think about this: https://now.prod.drupal.uiowa.edu/achievements? 

I took a first stab at building the Achievements page out and pulled in news tagged 'achievements' here. There aren’t many articles it looks like under this tag yet, so they all repeat on the page. Tried to make it similar to the Wisconsin example. It might need more here in terms of design, but not sure what that is…

**Comment by bspeare** [comment]
@mkyoder looks good.  Do you want to set those cards up as one news block using a three column grid?  Let me know if you need more clarification on that.

**Comment by mkyoder** [comment]
@bspeare Yes, could I get more clarification on that? That might be helpful. :)

**Comment by bspeare** [comment]
@mkyoder I updated the page and moved that block to a one column layout, and then site the list block to display as grid: https://now.prod.drupal.uiowa.edu/achievements.

**Comment by briand44** [comment]
Do we want to have a link from those pages to view all news or no?

**Comment by mkyoder** [comment]
The following pages are now created and linked:

- Featured Headlines 
- UI Spotlights 
- Achievements 
- In the news

**Comment by mkyoder** [comment]
@briand44 I can create one if needed. Just let me know.

**Comment by mkyoder** [comment]
Also, I know this needs to be removed right? There was something else with the main nav menu, but I can't remember what that was...
![Screen Shot 2023-01-27 at 3 48 29 PM](https://user-images.githubusercontent.com/107870104/215207873-337504b7-bf29-4604-be55-570492f0caba.png)

**Comment by briand44** [comment]
I kind of thought a link (in some fashion - call to action, more link, etc.) would be nice but I'll leave it up to you and Ben. I am not an expert, just a personal opinion.

**Comment by briand44** [comment]
Yeah, remove the Iowa Now link and we discussed the topics links, linking to the appropriate tags used on the existing site

**Comment by bspeare** [comment]
Do we need https://now.prod.drupal.uiowa.edu/experts-list added too?

**Comment by briand44** [comment]
If you want to check off the checkboxes in the original Issue Summary for what is complete, that should be a good guide of what is left as well.

**Comment by mkyoder** [comment]
Hmm.. I cannot seem to check those boxes on the issue summary. They are greyed out for me. Weird. 
![Screen Shot 2023-01-27 at 4 03 15 PM](https://user-images.githubusercontent.com/107870104/215212340-f9090c74-d826-4793-a858-fad0036c3110.png)

- Iowa Now removed from main menu nav
- Faculty and Staff added to main menu nav

**Comment by mkyoder** [comment]
> 

Okay, so these topics would be added to the main menu nav? 
![Screen Shot 2023-01-27 at 4 12 12 PM](https://user-images.githubusercontent.com/107870104/215214985-8117517e-98f4-420b-967b-8fcd85fdc9f7.png)

**Comment by bspeare** [comment]
@mkyoder Yes, under a "Topics" dropdown link that has `<nolink>` set on it.

**Comment by mkyoder** [comment]
Great! I added this as part of the main menu as a nolink set on it with the arts, campus, economic development, health care, impact, and research as dropdown pages. 

These are categories that are not added as tags yet (e.g. arts, impact, economic development), at least not that I saw when I checked. But might have missed them. Do you want me to add those as tags to pull in content on those pages? Is there content yet to pull in for those categories? I wasn’t sure.  

![Screen Shot 2023-02-06 at 3 57 30 PM](https://user-images.githubusercontent.com/107870104/217097357-d0adc229-b6fa-4204-b29a-70d9107d161a.png)

**Comment by mkyoder** [comment]
Added staff list page to main menu navigation, which links to osc.uiowa.edu based on feedback from Tricia via email. 
![Screen Shot 2023-02-06 at 3 57 39 PM](https://user-images.githubusercontent.com/107870104/217097465-8c7a7d56-a64e-4d1d-9de1-6c5783c273c2.png)

![Screen Shot 2023-02-06 at 3 58 06 PM](https://user-images.githubusercontent.com/107870104/217097499-d0e503d3-985b-4040-9c86-2856d02a2fb9.png)

**Comment by bspeare** [comment]
It looks like most of this is done.  Do we want to create a page for events (https://now.prod.drupal.uiowa.edu/events) or do we just link to events.uiowa.edu?  It seems like we might want to create a page on the site since this is pulling in custom tagged events.

**Comment by bspeare** [comment]
I wonder if we also want to move the social media stuff to the footer?  Since these are not Iowa Now specific social media accounts, it might make sense to have those in the footer like uiowa.edu.

**Comment by briand44** [comment]
>Do we want to create a page for events (https://now.prod.drupal.uiowa.edu/events) or do we just link to events.uiowa.edu? It seems like we might want to create a page on the site since this is pulling in custom tagged events.

Chris C. asked about how the event block works and I confirmed the current functionality. Maybe we follow up on that thread to see if that is what they want or if they prefer to link to a page with more? I think they will probably be ok with just three and linking out but that is just a guess.

**Comment by briand44** [comment]
Second thought... Let's just link to https://events.uiowa.edu/?k=IN-front-page for now and get feedback when we meet with them next

**Comment by briand44** [comment]
>I wonder if we also want to move the social media stuff to the footer? Since these are not Iowa Now specific social media accounts, it might make sense to have those in the footer like uiowa.edu.

I don't have strong feelings here. If we remove that section we may need to adjust some background colors.

**Comment by mkyoder** [comment]
Social media is now switched to footer

**Comment by mkyoder** [comment]
Background color of events section changed to white

**Comment by mkyoder** [comment]
Sorry, one more thing here. On these pages, do I need to create tags for them? 

![Screen Shot 2023-02-07 at 4 07 42 PM](https://user-images.githubusercontent.com/107870104/217379366-bfe55a15-2845-4a28-9aca-f0f709f56662.png)


Is there content to pull in here? Currently, I have it set to pull in other tagged items since those tags aren't created.

**Comment by briand44** [comment]
We shared this with our Iowa Now partners so going to close this out and open a new issue with the next round of changes
